src/components/MessageList/Message/index.tsx

```
@@ -13,31 +13,62 @@ import remarkGemoji from 'remark-gemoji';
-  const [isOpenDialog, setDialog] = useState(false);
+  const [isOpenDialog, setDialog] = useState<boolean>(false);
   const closeDialog = useCallback(() => {
     setDialog(false);
   }, []);
 
-  const { id, displayName, text, uid, timestamp, type, boardSize, image, gameBoardId } = message;
+  // Destructure common properties
+  const { id, displayName, text, uid, timestamp, type } = message;
+
+  // Then conditionally access type-specific properties
+  let boardSize, gameBoardId, image;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)